### DISCUSSION TITLE
Crazy Issues with GitHub and ATOM

### DISCUSSION THREAD

**Discussion by NicoSay0913** [atom/atom#22172] [Q&A]
Hello!

I'm a college student in Web Design and we are using ATOM with GitHub for homework assignments and for turning in the assignments for the teacher. Yesterday I noticed that all of my assignments were saving to this Desktop repository that I didn't remember making, and I could not change it to any of the new repositories I had been creating per assignment as told by my professor. Well, I stupidly decided to delete the repository....and that's when it all went downhill. 

No matter what I did, ATOM wouldn't recognize any other repository, nor would the Desktop repository name disappear from my ATOM account, so nothing would push through, it kept giving me errors and saying the branch did not exist, etc. 

Out of frustration, I ended up deleting my GitHub account, and uninstalling ATOM as cleanly as I could, hunting down all files possible and removing them. I then did a clean install and signed up for GitHub once again with a new email and user name, did the Git install in the command prompt screen again, etc. And still ATOM says the old account and refuses to push through to any other repository. 

My boyfriend (an IT man), spent a while going through the systems and researching, tried a bunch of stuff, and eventually was able to get my new GitHub account to show on ATOM, however, GitHub will now allow me to make other repositories, but ATOM will not change which one it uses, and it will not push through most times. 

I'm at a loss, I really just want to delete both things and start fresh where is works like it did before I got the stupid idea to delete a repository and apparently blew the whole thing up. 

Help would be greatly appreciated!

Thank you

**Comment by icecream17** [comment]
could you see if any errors pop up?
`ctrl+shift+i` then try to switch repos or do a commit

**Comment by king40or1** [comment]
Yes I have been experiencing this issue for years now, since I wasn't really an experienced developer I wasn't really sure how to address the issue.  I had to make this account recently because I haven't been able to access my main account.  Do you get a 404 error by chance on certain sites?

**Comment by ThatXliner** [comment]
You signed in to GitHub/Git, is that correct?

**Comment by icecream17** [comment]
I think this new issue: https://github.com/atom/atom/issues/22532 might be a start for working on issues like these. This time it's with deleting the folder instead of moving subfolder files around.
Also the https://github.com/atom/github repo is failing like crazy
